scss/_patterns_side-navigation.scss

```diff
@@ -163,3 +169,23 @@
     }
   }
 }
+
+@mixin vf-side-navigation-theme-light {
+  @include vf-side-navigation-theme(
+    $color-sidenav-text-default: $colors--light-theme--text-muted,
+    $color-sidenav-text-active: $colors--light-theme--text-default,
+    $color-sidenav-item-background-highlight: $colors--light-theme--background-highlighted,
+    $color-sidenav-item-border-highlight: $color-brand,
+    $color-sidenav-list-border: $colors--light-theme--border-default
+  );
+}
+
+@mixin vf-side-navigation-theme-dark {
+  @include vf-side-navigation-theme(
+    $color-sidenav-text-default: $colors--dark-theme--text-muted,
+    $color-sidenav-text-active: $colors--dark-theme--text-default,
+    $color-sidenav-item-background-highlight: $colors--dark-theme--background-highlighted,
+    $color-sidenav-item-border-highlight: $colors--dark-theme--text-default,
```

Comment:
@lyubomir-popov 
We wanted to use brand color for this and we do in light theme.

Shouldn't we do it as well in dark theme? Although in Vanilla brand color is dark grey, so it doesn't really work good in this case...

Reply:
I think the vanilla default should focus on being accessible. Any of these variables can be assigned the value of $color-brand.

Reply:
I don't think it can be easily overridden. If dark theme of side navigation uses `$colors--dark-theme--text-default` they would need to override text default to brand color to make border use that - but it will change all text in dark theme to brand color.

Maybe we need a separate color for that in every theme - like accent color? So it can be adjusted without affecting other color in a theme?

Reply:
@bartaz did you mean $color-sidenav-item-border-highlight? sorry got confused which colour we're talking about.

Reply:
just checked the tabstrip, it uses $color-mid-dark. Arguably, these should be tied together. Since they are not, I'd leave it as is at the moment and then we can do a pass and fixe everywhere if we decide we want the brand color.

Reply:
@lyubomir-popov `$color-sidenav-item-border-highlight` is a variable used inside of the `vf-side-navigation-theme` mixin. It's not a global setting, you can't change its value directly.

Currently in 'light' theme the value of this variable is `$color-brand`, so to change it, devs can change `$color-brand` value.
In dark theme we implemented it as `$colors--dark-theme--text-default`, so the only way to change it (when theme is dark) is to change `$colors--dark-theme--text-default` - but this will also affect all the text color in all the dark components.

If we want things like `$color-sidenav-item-background-highlight`, etc (every single theme color PER COMPONENT) exposed to be potentially changed individually (and not by changing shared 'theme' colors) it's a completely new feature that we currently don't have.

Not a hard thing to implement - we just need to expose these variables (with default values from theme colors), but the question is if we really need this level of granularity.
Because once we allow it and give it to users we will have to maintain it.